RLBotFramework/utils/item_id_parser.py

```diff
@@ -28,15 +34,14 @@
     Quality = line_parts[5]
     UnlockMethod = line_parts[6]
     #if it is platform specific, logo for season play, crate, bots, keys, packs
-    if (ID=='0' or AssetPackageName == 'seasonlogos' or
-    AssetPackageName == 'itemcontainer' or
-    AssetPackageName == 'bots' or
-    'key' in AssetPackageName or
-    'pack' in AssetPackageName):
+    if (ID=='0' or AssetPackageName == 'seasonlogos' or AssetPackageName == 'itemcontainer' or
+    AssetPackageName == 'bots' or 'key' in AssetPackageName or 'pack' in AssetPackageName):
         continue
+
     elif 'body' in FullName:
```

Comment:
Couldn't you make these all into using a dictionary?
Instead of doing the same thing?

Reply:
I actually thought of it, but I thought it would be harder to update.

Reply:
I agree with dtracers in that you could make a dictionary that contains all of this. Instead of individual arrays it could be `the_dict['Decals']` or `the_dict['Antennas']` so that you could just add a line later that adds a new type. Then the code below would automatically iterate over all the keys in the dictionary.